bulk-targeted-sponsorship/README.md

```diff
@@ -0,0 +1,381 @@
+
+_**For explanatory purposes only, this is a JavaScript function
+that calls the Ecosyste.ms Packages API for a single dependency.**_
+```JavaScript
+
+async function call_api_js(packageType, packageName) {
+    //// THE LINE BELOW CALLS THE FUNCTION BELOW.
+    const packageTypeChecked = await convertPackageTypeNaming(packageType)
+    const EcoSysPackages_baseURL = 'https://packages.ecosyste.ms/api/v1/registries/';
+    const EcoSysPackages_endpoint = '/packages/';
+    let url = `${EcoSysPackages_baseURL}${packageTypeChecked}${EcoSysPackages_endpoint}${packageName}`;
+    const data = [];
+    try {
+      const packagesEndpointData = await fetch(url).then((response) => {
+        return response.json();
+      })
+      return packagesEndpointData
+      }
+    catch (error) {
+        return `Request failed with error: ${error.message}`;
+    }
+}
+```
+
+
+#### Converting package types to ecosyste.ms registries 
+
+When working with dependency data in different systems, it is not uncommon for different systems to refer to 
+different package managers using slightly different strings. This is one example of a function that 
+takes a string that represents package managers and converts it into ecosyte.ms style
+naming conventions. You will likely have to make something similar.
+
+``` JavaScript
+function convertPackageTypeNaming(packageType){
+  const convertToEcosystems = {
+        "git":"github%20actions",
+        "npm": "npmjs.org",
+        "pypi": "pypi.org",
+        "pip": "pypi.org",
+        "nuGet": "nuget.org",
+        "nuget": "nuget.org",
+        "cargo":"crates.io",
+        "go":"proxy.golang.org",
+        "Go":"proxy.golang.org",
+        "Maven":"repo1.maven.org",
+        "rubyGem":'rubygems.org',
+        "rubygems":'rubygems.org',
+        "conda":'conda-forge.org',
+        "cocoapod":"cocoapod.org",
+         "vcpkg":"vcpkg.io",
+         "alpine":"alpine",
+         "clojars":"clojars,org",
+         "packagist":"packagist.org",
+         "bower":"bower.io"
+    } 
+    if (packageType in Object.keys(convertToEcosystems)){
+      return convertToEcosystems[packageType]
+    }
+    else {
+      return packageType
+    }
+}
+```
+
+## Filter dependencies to select a subset for funding
+### _Identifying how to have the most impact_
+
+If you've done what has been described above, you will have a listing of 
+your dependencies, information about how much you use each, and a
+mapping between your dependencies and metadata that describes those dependencies.
+The next step is to filter that list to target a subset of repositories that
+meet your minimum qualifications and a further sorting/filtering to identify
+the subset of dependencies that better match criteria you want to optimize for. 
+
+As example, you might want to absolutely not fund anything by your own employees
+and optimize for funding dependencies that see the most usage in your enterprise.
+The first is a binary filter. The second sorts the list.
```

Comment:
nit: this could be one sentence and improve flow.

Reply:
Combined